components/properties/src/props.rs

```diff
@@ -18,7 +18,7 @@ pub enum EnumeratedProperty {
     /// The East_Asian_Width property. See [`EastAsianWidth`].
     EastAsianWidth = 0x1004,
     /// The General Category property.
-    GeneralCategory = 0x1005,
+    GeneralCategoryGroup = 0x1005,
```

Comment:
In order to keep these parallel with ICU, it should be
- `GeneralCategory = 0x1005,` as for ICU4C unicode/uchar.h UCHAR_GENERAL_CATEGORY
- `GeneralCategoryGroup = 0x2000,` as for ICU4C unicode/uchar.h UCHAR_GENERAL_CATEGORY_MASK

(I used _MASK as for a bit mask in some places, but some API docs also use "group" like in this PR and like UAX 44's "groupings".)